docs/manual/docs/cmd/outlook/report/report-getmailboxusagestorage.md

```diff
@@ -0,0 +1,40 @@
+# outlook report getmailboxusagestorage
+
+Get the amount of storage used in your organization. 
+
+## Usage
+
+```sh
+outlook report getmailboxusagestorage [options]
+```
+
+## Options
+
+Option|Description
+------|-----------
+`--help`|output usage information
+`-p, --period <period>`|The length of time over which the report is aggregated. Supported values `D7|D30|D90|D180`
+`-f, --outputFile [outputFile]`|Path to the file where the report should be stored in
+`-o, --output [output]`|Output type. `text|json`. Default `text`
+`--verbose`|Runs command with verbose logging
+`--debug`|Runs command with debug logging
+
+## Examples
+
+Gets the amount of storage used in your organization for the last week
```

Comment:
Could you add `mailbox` in front of storage in the examples? It might be a bit more clear for noobs like me. Thanks!